lib/middleware.js

```diff
@@ -33,10 +33,12 @@ module.exports = {
 
   writeHtmlFile(req, res, next) {
     debug("write html file");
-    if (!req.body.html)
-      next({ error: "Request must contain HTML for conversion" });
+    if (!req.body.html) {
+      return next(new Error("Request must contain HTML for conversion"));
```

Comment:
If we want to actually stop execution in error cases, we need to `return` otherwise we hand off to the error handler but the execution of this function will continue.

Anywhere that we're trying to use a guard clause like this we need to `return` as far as I understand it.